test/js/components/projects/detail.test.jsx

```diff
@@ -134,9 +134,9 @@ const defaultState = {
           status: 'Review',
         },
       ],
-      next: 'next-url',
+      next: 'next-epic-url',
       notFound: [],
-      fetched: ['all'],
+      fetched: true,
```

Comment:
These are part of the epic reducer (I know, the naming here is too confusing!), so `fetched` is still a boolean.